.github/workflows/release-jetbrains.yml

```diff
@@ -0,0 +1,80 @@
+name: release-jetbrains
+on:
+  push:
+    tags:
+      - "jetbrains/v*"
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  release-jetbrains:
+    name: release JetBrains plugin
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      # Set up Java environment for the next steps
+      - uses: actions/setup-java@v4
+        with:
+          distribution: zulu
+          java-version: 17
+      # Setup Gradle
+      - uses: gradle/actions/setup-gradle@v3
+        with:
+          gradle-home-cache-cleanup: true
+      # Set environment variables
+      - name: Export Properties
+        id: properties
+        shell: bash
+        run: |
+          CHANGELOG="$(cat << 'EOM' | sed -e 's/^[[:space:]]*$//g' -e '/./,$!d'
+          ${{ github.event.release.body }}
+          EOM
+          )"
+          
+          echo "changelog<<EOF" >> $GITHUB_OUTPUT
+          echo "$CHANGELOG" >> $GITHUB_OUTPUT
+          echo "EOF" >> $GITHUB_OUTPUT
+      # Publish the plugin to JetBrains Marketplace
+      - name: Publish Plugin
+        env:
+          PUBLISH_TOKEN: ${{ secrets.JET_BRAINS_PUBLISH_TOKEN }}
+        run: ./gradlew publishPlugin
+
+      # Upload artifact as a release asset
+      - name: Upload Release Asset
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: gh release upload ${{ github.event.release.tag_name }} ./build/distributions/*
+
+      # Create a pull request
+      - name: Create Pull Request
```

Comment:
Why create a pull request ?

Reply:
The workflow [updates the changelog](https://github.com/JetBrains/intellij-platform-plugin-template/blob/9726f523c5a41e275fc1fd9f52e01a885ad8e93f/.github/workflows/release.yml#L45-L60) based on the release body, so it needs to update the original changelog. Even though it is nice to have the changelog, I think we don't really need it since it will contain many unrelated changes. Let me remove it 🙆‍♂️